conf/establishersCompanyDirector.routes

```diff
@@ -0,0 +1,164 @@
+
+GET        /:establisherIndex/company/director/:directorIndex/director-name                                         controllers.register.establishers.company.director.DirectorNameController.onPageLoad(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/director-name                                         controllers.register.establishers.company.director.DirectorNameController.onSubmit(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+GET        /:establisherIndex/company/director/:directorIndex/change/director-name                                  controllers.register.establishers.company.director.DirectorNameController.onPageLoad(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/change/director-name                                  controllers.register.establishers.company.director.DirectorNameController.onSubmit(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+GET        /:establisherIndex/company/director/:directorIndex/:mode/director-name/:srn                              controllers.register.establishers.company.director.DirectorNameController.onPageLoad(mode: Mode, establisherIndex: Index, directorIndex: Index, srn: Option[String])
+POST       /:establisherIndex/company/director/:directorIndex/:mode/director-name/:srn                              controllers.register.establishers.company.director.DirectorNameController.onSubmit(mode: Mode, establisherIndex: Index, directorIndex: Index, srn: Option[String])
+
+GET        /:establisherIndex/company/director/:directorIndex/director-date-of-birth                                controllers.register.establishers.company.director.DirectorDOBController.onPageLoad(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/director-date-of-birth                                controllers.register.establishers.company.director.DirectorDOBController.onSubmit(mode: Mode = NormalMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+GET        /:establisherIndex/company/director/:directorIndex/change/director-date-of-birth                         controllers.register.establishers.company.director.DirectorDOBController.onPageLoad(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
+POST       /:establisherIndex/company/director/:directorIndex/change/director-date-of-birth                         controllers.register.establishers.company.director.DirectorDOBController.onSubmit(mode: Mode = CheckMode, establisherIndex: Index, directorIndex: Index, srn: Option[String] = None)
```

Comment:
DirectorDOB routes are needed in UpdateMode

Reply:
done